Pinta.Core/Managers/LivePreviewManager.cs

```diff
@@ -70,172 +69,126 @@ public async void Start (BaseEffect effect)
 		if (IsEnabled)
 			throw new InvalidOperationException ("LivePreviewManager.Start() called while live preview is already enabled.");
 
-		string effectName = effect.Name;
-
-		// Create live preview surface.
-		// Start rendering.
-		// Listen for changes to effectConfiguration object, and restart render if needed.
+		// Handle selection path.
```

Comment:
I think this comment went along with the code (now further down) for initializing the render bounds from the selection. It's not a particularly useful comment though